arangodb-net-standard/ViewApi/Models/ViewDetails.cs

```diff
@@ -0,0 +1,115 @@
+
+        /// <summary>
+        /// The number of commits between 
+        /// removing unused files in
+        /// the ArangoSearch data directory 
+        /// (default: 2, to disable use: 0).
+        /// Read more about this in the documentation.
+        /// </summary>
+        public int CleanupIntervalStep { get; set; }
+
+        /// <summary>
+        /// The number of milliseconds to wait
+        /// between committing View data store 
+        /// changes and making documents visible
+        /// to queries (default: 1000, to disable use: 0)
+        /// Read more about this in the documentation.
+        /// </summary>
+        public int CommitIntervalMsec { get; set; }
+
+        /// <summary>
+        /// The number of milliseconds to wait
+        /// between applying ‘consolidationPolicy’ 
+        /// to consolidate View data store and 
+        /// possibly release space on the filesystem
+        /// (default: 10000, to disable use: 0). 
+        /// Read more about this in the documentation.
+        /// </summary>
+        public int ConsolidationIntervalMsec { get; set; }
+
+        /// <summary>
+        /// The consolidation policy to apply
+        /// for selecting which segments should be merged.
+        /// Read more about this in the documentation.
+        /// </summary>
+        public ViewConsolidationPolicy ConsolidationPolicy { get; set; }
+
+        /// <summary>
+        /// A primary sort order can be defined 
+        /// to enable an AQL optimization.
+        /// Read more about this in the documentation.
+        /// </summary>
+        public List<ViewSort> PrimarySort { get; set; }
+
+        /// <summary>
+        /// Defines how to compress the primary sort data 
+        /// (introduced in v3.7.1). ArangoDB v3.5 and v3.6 
+        /// always compress the index using LZ4. 
+        /// This option is immutable. Possible values:
+        /// 1) "lz4" (default): use LZ4 fast compression.
+        /// 2) "none": disable compression to trade space for speed.
+        /// Read more about this in the documentation.
+        /// </summary>
+        public string PrimarySortCompression { get; set; }
+
+        /// <summary>
+        /// An array of objects to describe which document 
+        /// attributes to store in the View index (introduced in v3.7.1).
+        /// It can then cover search queries, which means 
+        /// the data can be taken from the index directly
+        /// and accessing the storage engine can be avoided.
+        /// </summary>
+        public List<ViewStoredValue> StoredValues { get; set; }
+
+        /// <summary>
+        /// Maximum number of concurrent active
+        /// writers (segments) that perform a transaction. 
+        /// Other writers (segments) wait till current 
+        /// active writers (segments) finish 
+        /// (default: 0, use 0 to disable, immutable)
+        /// </summary>
+        public int? WritebufferActive { get; set; }
```

Comment:
A minor thing: these properties are not optional when returned by `GET /_api/view/{view-name}/properties`. We could separate the request/response models to make the distinction. Though callers may find it convenient to be able to use the same type for both the request and response, so maybe it's better as is. Is that what you had in mind when using the same model for request/response?

Reply:
Yes, the idea was that they would use the same model in the request and response.